anyway/utilities.py

```diff
@@ -180,9 +180,9 @@ def fetch_first_and_every_nth_value_for_column(conn, column_to_fetch, n):
     select_query = select([sub_query]) \
         .where(or_(func.mod(sub_query.c.row_number, n) == 0,
                    sub_query.c.row_number == 1))
-    ids_and_row_numbers = conn.execute(select_query).fetchall()
-    ids = [id_and_row_number[0] for id_and_row_number in ids_and_row_numbers]
-    return ids
+    cursor = conn.execution_options(stream_results=True).execute(select_query)
```

Comment:
from what I understand from the docs, stream_results=True will help only if we use psycopg2, is that what we use? 

https://docs.sqlalchemy.org/en/13/core/connections.html

Reply:
Yes 👍🏽